LSPS6/README.md

```diff
@@ -0,0 +1,130 @@
+The LSP SHOULD serve a well-known URI `/.well-known/bitcoin` to allow for public key discovery.
+
+```json
+{
+  "icon": "https://acme.com/icon.png",
+  "logo": "https://acme.com/logo.png",
+  "email": "support@acme.com",
+  "nostr": "npub1w...s2mfp9",
+  "twitter": "@acme",
+  "telegram": "@acme",
+  "lightning": {
+    "mainnet": {
+      "public_keys": [
+        "0326e6...2833a3",
+        "0305f5...2e4e45"
+      ]
+    },
+    "testnet": {
+      "public_keys": [
+        "03f060...869c00"
+      ]
+    },
+    ...
+  }
+}
+```
+
+- `icon` LSP MAY include the URL of its icon.
+- `logo` LSP MAY include the URL of its logo.
+- `email` LSP MAY include its support email.
+- `nostr` LSP MAY include its nost public key.
+- `twitter` LSP MAY include its Twitter handle.
+- `telegram` LSP MAY include its Telegram handle.
+- `lightning`
+  - `mainnet` LSP SHOULD include its mainnet lightning node public keys.
+  - `testnet` LSP MAY include its testnet lightning node public keys.
+  - `...` LSP MAY include its public keys for other network types.
+
+[Well-known URIs]: https://datatracker.ietf.org/doc/html/rfc8615
+
+## API
+
+### lsps6.get_info
+
+`lsps6.get_info` provides generic information about the LSP.
+
+The client SHOULD use the data in `url` up until the [TLD][] and append `/.well-known/bitcoin`.
+So assuming the URL is `https://acme.com/lsp`, the client MUST use `https://acme.com/.well-known/bitcoin`.
+To confirm authority the client MUST crawl this URL and
+match the public key from the LSP with the public key from the well-known URI.
+
+[TLD]: https://en.wikipedia.org/wiki/Top-level_domain
+
+Any information that is available in the response of `lsps6.get_info`
+should override the information in the well-known URI as it's more authoritative.
+
+If there is a piece of information unavailable,
+the client SHOULD use the information from the well-known URI as a fallback.
+
+**Request** No parameters needed.
+
+**Response**
+
+```JSON
+{
+  "url": "https://acme.com/lsp",
+  "icon": "https://acme.com/lsp-icon.png",
+  "logo": "https://acme.com/lsp-logo.png",
+  "email": "lsp-support@acme.com",
+  "nostr": "npub1w...s2mfp9",
+  "twitter": "@acme-lsp",
+  "telegram": "@acme-lsp"
+}
+```
+
+- `url` LSP SHOULD include the URL of its website.
+- `icon` LSP MAY include the URL of its icon.
+- `logo` LSP MAY include the URL of its logo.
+- `email` LSP MAY include its support email.
+- `nostr` LSP MAY include its nost public key.
+- `twitter` LSP MAY include its Twitter handle.
+- `telegram` LSP MAY include its Telegram handle.
+
+### lsps6.get_lsp_info
+
+`lsps6.get_lsp_info` provides generic information about an LSPS spec.
+
+**Request**
+```JSON
+{
+  "spec": "LSPS1"
+}
+```
+
+**Response**
+
+```JSON
+{
+  "short": "Short description",
+  "long": "Longer description of the details"
+}
+```
+
+LSP MUST always respond even when there is no support for a spec. LSP MUST include the following fields
```

Comment:
Why always respond? Why not define an "NOT_FOUND" or "UNSUPPORTED" error?

Reply:
I didn't want to duplicate logic in this LSPS with differentiating if you support an LSPS or not. You have LSPS1 for that? All this LSPS does is provide a way to confirm authenticity and optionally add meta-data?

Reply:
One more reason to put such method in their specific LSPS :)